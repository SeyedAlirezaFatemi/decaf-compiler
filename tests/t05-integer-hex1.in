0x0
0x1
0xe43620
0x11
0xbeef
0xF
0xF00
0xB1ad
