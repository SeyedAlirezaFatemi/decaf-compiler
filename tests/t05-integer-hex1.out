T_INTLITERAL 0x0
T_INTLITERAL 0x1
T_INTLITERAL 0xe43620
T_INTLITERAL 0x11
T_INTLITERAL 0xbeef
T_INTLITERAL 0xF
T_INTLITERAL 0xF00
T_INTLITERAL 0xB1ad
